drv/bl_radio/bl_radio_default.c

```diff
@@ -32,34 +32,36 @@
 
-#define RADIO_SHORTS_COMMON (RADIO_SHORTS_END_DISABLE_Enabled << RADIO_SHORTS_END_DISABLE_Pos) | \
-                            (RADIO_SHORTS_ADDRESS_RSSISTART_Enabled << RADIO_SHORTS_ADDRESS_RSSISTART_Pos) | \
-                            (RADIO_SHORTS_DISABLED_RSSISTOP_Enabled << RADIO_SHORTS_DISABLED_RSSISTOP_Pos)
+#define RADIO_SHORTS_COMMON (RADIO_SHORTS_END_DISABLE_Enabled << RADIO_SHORTS_END_DISABLE_Pos) |                 \
+                                (RADIO_SHORTS_ADDRESS_RSSISTART_Enabled << RADIO_SHORTS_ADDRESS_RSSISTART_Pos) | \
+                                (RADIO_SHORTS_DISABLED_RSSISTOP_Enabled << RADIO_SHORTS_DISABLED_RSSISTOP_Pos)
 
-#define RADIO_INTERRUPTS    (RADIO_INTENSET_ADDRESS_Enabled << RADIO_INTENSET_ADDRESS_Pos) | \
-                            (RADIO_INTENSET_END_Enabled << RADIO_INTENSET_END_Pos) | \
-                            (RADIO_INTENSET_DISABLED_Enabled << RADIO_INTENSET_DISABLED_Pos)
+#define RADIO_INTERRUPTS (RADIO_INTENSET_ADDRESS_Enabled << RADIO_INTENSET_ADDRESS_Pos) | \
+                             (RADIO_INTENSET_END_Enabled << RADIO_INTENSET_END_Pos) |     \
+                             (RADIO_INTENSET_DISABLED_Enabled << RADIO_INTENSET_DISABLED_Pos)
 
 #define RADIO_STATE_IDLE 0x00
-#define RADIO_STATE_RX   0x01
-#define RADIO_STATE_TX   0x02
+#define RADIO_STATE_RX 0x01
+#define RADIO_STATE_TX 0x02
 #define RADIO_STATE_BUSY 0x04
 
-typedef struct __attribute__((packed)) {
-    uint8_t header;                              ///< PDU header (depends on the type of PDU - advertising physical channel or Data physical channel)
-    uint8_t length;                              ///< Length of the payload + MIC (if any)
-    uint8_t payload[BL_BLE_PAYLOAD_MAX_LENGTH];  ///< Payload + MIC (if any) (BL_BLE_PAYLOAD_MAX_LENGTH > BL_IEEE802154_PAYLOAD_MAX_LENGTH)
+typedef struct __attribute__((packed))
+{
+    uint8_t header;                             ///< PDU header (depends on the type of PDU - advertising physical channel or Data physical channel)
+    uint8_t length;                             ///< Length of the payload + MIC (if any)
+    uint8_t payload[BL_BLE_PAYLOAD_MAX_LENGTH]; ///< Payload + MIC (if any) (BL_BLE_PAYLOAD_MAX_LENGTH > BL_IEEE802154_PAYLOAD_MAX_LENGTH)
 } radio_pdu_t;
 
-typedef struct {
-    radio_pdu_t     pdu;       ///< Variable that stores the radio PDU (protocol data unit) that arrives and the radio packets that are about to be sent.
-    bool            pending_rx_read; ///< Flag to indicate that a PDU has been received, but not yet read by the application.
-    radio_ts_packet_t start_pac_cb;  ///< Function pointer, stores the callback to capture the start of the packet.
-    radio_ts_packet_t end_pac_cb;      ///< Function pointer, stores the callback to capture the end of the packet.
-    uint8_t         state;     ///< Internal state of the radio
-    bl_radio_mode_t mode;      ///< PHY protocol used by the radio (BLE, IEEE 802.15.4)
+typedef struct
+{
```

Comment:
Please do not use the convention of having `{` in the next line (the whole codebase uses same-line `{`).